validator.go

```diff
@@ -62,34 +63,44 @@ type Validator interface {
 	GetResponseBodyValidator() responses.ResponseBodyValidator
 }
 
+type Option func(*validator)
+
+func WithRegexEngine(engine jsonschema.RegexpEngine) Option {
+	return func(v *validator) {
+		v.regexEngine = engine
+	}
+}
```

Comment:
Do they have to be exported?

I rgi k it's the idea of your PR to allow to overload, but I prefer asking in case the scope could be reduced

Reply:
I believe they do need to be exported as the 'New' functions that they're associated with are exported

Reply:
indeed